src/components/userSettings/UserSettings.tsx

```diff
@@ -12,11 +12,14 @@ import { useAccount } from 'hooks';
 import { Button } from 'primaries/SharedStyling';
 import { ImageV2 } from 'components/reusables/SharedStylingV2';
 import { updateBulkSubscriptions, updateBulkUserSettings } from 'redux/slices/channelSlice';
+import UserProfileSettings from 'components/UserProfileSettings/UserProfileSettings';
```

Comment:
When i click on the upload image for the avatar in setting page. it throws validation error for name field.